config/default/filter.format.minimal_html.yml

```diff
@@ -19,7 +19,7 @@ filters:
     status: true
     weight: -10
     settings:
-      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <hr> <p class=""> <span class=""> <table class=""> <br> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class=" hs-button hs-button--big hs-secondary-button hs-more-link hs-external-link hs-file-link hs-pdf-link hs-private-link">'
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <hr> <p class> <span class> <table class> <br> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class=" hs-button hs-button--big hs-secondary-button hs-more-link hs-external-link hs-file-link hs-pdf-link hs-private-link">'
```

Comment:
I hate that it does this. You'll have to remove any `=""` from this change.

Reply:
I only see one and it's only a single quote - ```class="``` is that ```="``` what you would like removed?

Reply:
Oh nevermind, I think I read the diff wrong. This actually fixes an issue.